### PR TITLE
[release-v1.1] Add patches and fix generate release

### PR DESCRIPTION
**Pull request by pierDipi**
Same as https://github.com/openshift-knative/eventing-kafka-broker/pull/116 but for v1.1 branch

### CODE REVIEW

**Comment by pierDipi** [comment]
```

ERRO[2022-02-15T09:55:05Z] 
  * could not run steps: step conformance-aws-ocp-49 failed: failed to wait for the created cluster claim to become ready: timed out waiting for the condition 
```
/test 49-conformance-aws-ocp-49

File: openshift/release/knative-eventing-kafka-broker-dp-ci.yaml [hunk 1]
```diff
@@ -126,14 +126,14 @@ spec:
...
-        kafka.eventing.knative.dev/release: devel
+        kafka.eventing.knative.dev/release: v1.1.0
     spec:
       serviceAccountName: knative-kafka-broker-data-plane
       securityContext:
         runAsNonRoot: false
       containers:
         - name: kafka-broker-dispatcher
-          image: registry.ci.openshift.org/openshift/knative-v1.1.3:knative-eventing-kafka-broker-dispatcher
+          image: registry.ci.openshift.org/openshift/knative-v1.1.0:knative-eventing-kafka-broker-dispatcher
```
Comment (matzew) [inline openshift/release/knative-eventing-kafka-broker-dp-ci.yaml:136]: we are actually using 1.1.3 as the promotion:
https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/eventing-kafka-broker/openshift-knative-eventing-kafka-broker-release-v1.1__49.yaml#L163

**Review by matzew — CHANGES_REQUESTED** [review]
image versions are wrong

File: openshift/release/knative-eventing-kafka-broker-dp-ci.yaml (hunk 1 above)
Comment (pierDipi) [reply openshift/release/knative-eventing-kafka-broker-dp-ci.yaml:136]: Ok, thanks for helping me connect the dots.

would be better (easier to understand) make the promotion `knative-v1.1.0`?
For example, in other components, we don't care about the patch version.

Comment (matzew) [reply openshift/release/knative-eventing-kafka-broker-dp-ci.yaml]: We usually do only care if there is upstream manifest changes in a `dot release`.

I can't exactly recall any specific reason, but I used the latest `1.1.x` when starting, and so nailed my WIP pr for S-O, to exactly that 1.1.3 version

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**